recipe/meta.yaml

```diff
@@ -18,8 +18,8 @@ requirements:
     - cartopy >=0.14
     - cf_units >=2
     - cftime
-    - dask >=0.17.1
-    - numpy
+    - dask
+    - numpy >=1.14, !=1.15.2
```

Comment:
Why not 1.15.2? If you are worried about the py37 issue I think we have the backport patch already.

Reply:
What py37 issue?  We were having problems with build number 1 of numpy 1.15.2.  Is that related?

Reply:
@ocefpaf see https://travis-ci.org/SciTools/iris/jobs/439687656 for an example of the build problem seemingly being caused by NumPy v1.15.2 build 1

Reply:
> What py37 issue? 

This one: https://github.com/numpy/numpy/pull/12062

> @ocefpaf see travis-ci.org/SciTools/iris/jobs/439687656 for an example of the build problem seemingly being caused by NumPy v1.15.2 build 1

Thanks @dkillick! That was fixed as well. Can you retry the CIs and test again?